README.md

```diff
@@ -1 +1,27 @@
-# jobsite
\ No newline at end of file
+# jobsite
```

Comment:
so GOOD! very clear, and I love the hotloading option

Reply:
Thank you @sbarkerdudley was the lead on hotloading.